From: Ben Hutchings <ben@decadent.org.uk>
Subject: aufs: mark as staging
Forwarded: not-needed

This hasn't been reviewed upstream, so should have the same status
as code in drivers/staging.

Also note in the MAINTAINERS file that it's not upstream.

--- a/fs/aufs/module.c
+++ b/fs/aufs/module.c
@@ -103,6 +103,7 @@ MODULE_DESCRIPTION(AUFS_NAME
 	" -- Advanced multi layered unification filesystem");
 MODULE_VERSION(AUFS_VERSION);
 MODULE_ALIAS_FS(AUFS_NAME);
+MODULE_INFO(staging, "Y");
 
 /* this module parameter has no meaning when SYSFS is disabled */
 int sysaufs_brs = 1;
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -1870,7 +1870,7 @@ L:	linux-unionfs@vger.kernel.org
 L:	aufs-users@lists.sourceforge.net (members only)
 W:	http://aufs.sourceforge.net
 T:	git://github.com/sfjro/aufs4-linux.git
-S:	Supported
+S:	Supported, not upstream
 F:	Documentation/filesystems/aufs/
 F:	Documentation/ABI/testing/debugfs-aufs
 F:	Documentation/ABI/testing/sysfs-aufs
